etc/config/.env.example

```diff
@@ -54,4 +54,9 @@ MODULE_WHITELIST=Magento_Framework,ConfigurableProductWishlist,ConfigurableProdu
 
 #*** Default timeout for wait actions
 #WAIT_TIMEOUT=10
+
+#*** Uncomment and set to enable browser log entries on actions in Allure. Blacklist is used to filter logs of a specific "source"
+#ENABLE_BROWSER_LOG=true
+#BROWSER_LOG_BLACKLIST=other
```

Comment:
`console-api` is not in the blacklist, is this on purpose?

Reply:
Kind of, yes. `other` maps to miscelaneous Chrome specific warnings as far as I can tell, which wouldn't be particularly useful.

Do you think thi sblacklist should be blak by default?

Reply:
If it's on purpose, I am fine. No need to change.